pkg/destroy/aws/aws.go

```diff
@@ -500,6 +505,11 @@ func getEC2Client(awsSession *session.Session) *ec2.EC2 {
 	return ec2.New(awsSession)
 }
 
+// getIAMClient is a wrapper for creating an AWS IAM client.
+func getIAMClient(awsSession *session.Session) *iam.IAM {
+	return iam.New(awsSession)
```

Comment:
Why bother with these wrapper functions?  `getIAMClient` is more characters than the code it wraps ;).  But we also have some existing precedent for these in `getEC2Client`, so I'm fine either way and can clean these up later.

Reply:
Just following the convention that was there but you make a good point. :)